src/main/java/spoon/reflect/visitor/ImportScannerImpl.java

```diff
@@ -224,11 +186,7 @@ public void scan(CtElement element) {
 
 	@Override
 	public Collection<CtTypeReference<?>> computeImports(CtElement element) {
-		classImports.clear();
-		fieldImports.clear();
-		methodImports.clear();
```

Comment:
This one is used to for `toString` so it can really be used multiple times with the same instance, it would be safer to keep it that way, don't you think?